src/App.tsx

```diff
@@ -20,10 +28,24 @@ function App() {
       ]),
     )
   return (
-    <div>
-      <button onClick={handleClick}>click</button>
-      <Body />
-    </div>
+    <Layout>
+      <>
+        <button onClick={handleClick}>click</button>
+        <Tabs orientation='vertical' height='80vh' overflowY='scroll'>
+          <TabList>
+            {Object.keys(data).map((key) => (
+              <Tab key={key}>{key}</Tab>
+            ))}
+          </TabList>
```

Comment:
This is clever🙂 I'd have written manually. Are you sure the response's object keys are in the same order all the time? If so that's great!

Reply:
@kei95 I checked https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys but not sure the Object.keys() always return an array with the same order. 
However I think we can guarantee the order consistency by using the array sort method as Object.keys() returns an array. Not sure if that operation is readable though.